README.md

```diff
@@ -10,7 +10,7 @@
 [![Dependency Status](https://david-dm.org/SAP/ui5-project/master.svg)](https://david-dm.org/SAP/ui5-project/master)
 [![devDependency Status](https://david-dm.org/SAP/ui5-project/master/dev-status.svg)](https://david-dm.org/SAP/ui5-project/master#info=devDependencies)
 
-**This is an alpha release!**  
+**This is an alpha release!**
```

Comment:
Trimming white spaces in markdown files is no good. Two blanks create a line break. This one just got lost

Reply:
Done